### PR TITLE
Output changes made.

### PR DESCRIPTION
**Pull request by ssailor96**
Number of output files reduced from 2 to 1. Duplicate file names and number of duplicates are now printed to the console.

### CODE REVIEW

File: duplicate_checker.py [hunk 1]
```diff
@@ -103,47 +103,36 @@ def dupeFinder(fileList):
...
         outfile.write(json_string)
     outfile.close()
 
+    # check if dupeList is empty
+    if not dupeList:
+        print("********************* No duplicate files found *********************")
+    else:
+        print("********************* Duplicate files found! Duplicates listed below *********************")
+        # print all duplicates to console along with the number of duplicates
+        numDuplicates = 0
```
Comment (ssailor) [inline duplicate_checker.py:126]: I would use the len() method to print the number of duplicates. https://www.w3schools.com/python/ref_func_len.asp

File: duplicate_checker.py [hunk 2]
```diff
@@ -103,47 +103,36 @@ def dupeFinder(fileList):
...
-def writeMetadata(fileList):
     # clean up datetime string to remove spaces and colons
     currentTime = str(datetime.now())
     currentTime = currentTime.replace(" ", "--")
     currentTime = currentTime.replace(":", "-")
 
-    mdFileName = "metadata_" + currentTime + ".json"
-    # mdFileName = "metadata_" + str(datetime.now()) + ".json"
+    # create file name with current datetime
+    outputFileName = "metadata_" + currentTime + ".json"
```
Comment (ssailor) [inline duplicate_checker.py]: I recommend giving the output files a more meaningful name such as hash_report-xxxxx. I also recommend reformating the date that is generated to remove the periods. You can use the strftime method on a DateTime to print a format. I recommend using the format strftime(%Y-%m-%dT%H-%M-%S"). Using this I would expect the output name to be something like hash_report-2022-09-28T14-14-22.json.

**Comment by ssailor** [comment]
You should add an entry to your .gitignore to ensure that output files you generate locally don't get checked into the repo. If you don't expect to have any json files that should be checked into the repo you can `*.json` to your .gitignore file and they will be automatically excluded from checkin.

**Review by ssailor — CHANGES_REQUESTED** [review]

File: duplicate_checker.py (hunk 1 above)
Comment (ssailor96) [reply duplicate_checker.py:126]: Cannot use length function in this case, I am finding duplicates using the isDuplicate attribute.

File: duplicate_checker.py (hunk 2 above)
Comment (ssailor96) [reply duplicate_checker.py]: The output file name format has been updated.

File: duplicate_checker.py (hunk 1 above)
Comment (ssailor) [reply duplicate_checker.py:126]: That makes sense, sorry I missed that.

File: duplicate_checker.py (hunk 2 above)
Comment (ssailor) [reply duplicate_checker.py]: I ran this and can confirm, the output JSON filename is now what I expected.

**Review by ssailor — APPROVED** [review]
Looks good now. Go ahead and merge this PR into dev

**Pull request merged**